### PR TITLE
Spoon, ParametrizedTest

### PR DESCRIPTION
**Pull request by moozo89**
Added Spoon
Added simple Parametrized Test
Refactor

### CODE REVIEW

File: Aplikacja/app/src/androidTest/java/pl/lizardproject/qe2016/AddItemParamTest.java [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+    @Before
+    public void before(){
+        itemListPage = new ItemListPage();
+    }
+
+    @Test
+    public void addItem() {
+        AddItemPage addItemPage;
+
+            addItemPage = itemListPage.goToAddItemPage();
```
Comment (mgorski-zlatan) [inline Aplikacja/app/src/androidTest/java/pl/lizardproject/qe2016/AddItemParamTest.java]: please format the code :)

File: Aplikacja/app/build.gradle [hunk 2]
```diff
@@ -2,6 +2,22 @@ apply plugin: 'com.android.application'
 apply plugin: 'kotlin-android'
 apply plugin: 'kotlin-android-extensions'
 
+buildscript {
+    repositories {
+        mavenCentral()
+    }
+
+    dependencies {
+        classpath 'com.stanfy.spoon:spoon-gradle-plugin:1.2.0'
```
Comment (mgorski-zlatan) [inline Aplikacja/app/build.gradle]: move classpath to main build.gradle and remove whole buildscript from this file.

File: Aplikacja/app/src/main/kotlin/pl/lizardproject/qe2016/edititem/EditItemViewModel.kt [hunk 3]
```diff
@@ -25,6 +25,9 @@ class EditItemViewModel(val itemId: Int?, private val activity: Activity, privat
     val priorities = Priority.values().map { it.toString().toLowerCase() }
 
     fun saveItemCommand(view: View) {
+        if(newItemName.get() == "crash"){
```
Comment (mgorski-zlatan) [inline Aplikacja/app/src/main/kotlin/pl/lizardproject/qe2016/edititem/EditItemViewModel.kt]: for now please remove it :)

**Comment by moozo89** [comment]
done

**Pull request merged**